### DISCUSSION TITLE
Make Jazzy CI job required.

### DISCUSSION THREAD

**Discussion by sea-bass** [RobotWebTools/rosbridge_suite#950] [General]
I just auto-merged a PR despite there being a failure in the Jazzy CI action.

Turns out this job is not marked as required in this repo.

@MatthijsBurgh can you set this up?

**Comment by MatthijsBurgh** [comment]
Jazzy is now required. Please fix the CI before making any other changes to this repo.

**Reply by sea-bass** [reply]
Thanks! It seemed to have been a flaky testz but I will keep a look out.
